enumerable.rb

```diff
@@ -0,0 +1,89 @@
+module Enumerable
+  def my_each
+    return to_enum unless block_given?
```

Comment:
### **my_each** 

- [ ] When a block is given after calling the given block once for each element in self, returns the array itself.

Reply:
fixed